src/test/java/com/powsybl/openloadflow/OpenSecurityAnalysisTest.java

```diff
@@ -0,0 +1,213 @@
+                .setNode1(5)
+                .setVoltageLevel2("VL2")
+                .setNode2(1)
+                .setR(3.0)
+                .setX(33.0)
+                .setG1(0.0)
+                .setB1(386E-6 / 2)
+                .setG2(0.0)
+                .setB2(386E-6 / 2)
+                .add();
+
+        network.newLine()
+                .setId("L2")
+                .setVoltageLevel1("VL1")
+                .setNode1(3)
+                .setVoltageLevel2("VL2")
+                .setNode2(2)
+                .setR(3.0)
+                .setX(33.0)
+                .setG1(0.0)
+                .setB1(386E-6 / 2)
+                .setG2(0.0)
+                .setB2(386E-6 / 2)
+                .add();
+
+        network.getLine("L1").newCurrentLimits1().setPermanentLimit(300000.0).add();
+        network.getLine("L1").newCurrentLimits2().setPermanentLimit(300000.0).add();
+        network.getLine("L2").newCurrentLimits1().setPermanentLimit(300000.0).add();
+        network.getLine("L2").newCurrentLimits2().setPermanentLimit(300000.0).add();
+
+        return network;
+    }
+
+    @BeforeEach
+    void setUp() {
+        network = createNetwork();
+    }
+
+    @Test
+    void test() {
+        SecurityAnalysisParameters saParameters = new SecurityAnalysisParameters();
+        LoadFlowParameters lfParameters = new LoadFlowParameters();
+        OpenLoadFlowParameters olfParameters = new OpenLoadFlowParameters()
+                .setSlackBusSelector(new NameSlackBusSelector("VL1_1"));
+        lfParameters.addExtension(OpenLoadFlowParameters.class, olfParameters);
+        saParameters.setLoadFlowParameters(lfParameters);
+        ContingenciesProvider contingenciesProvider = network -> Arrays.asList(new Contingency("L1", new BranchContingency("L1")),
+                                                                               new Contingency("L2", new BranchContingency("L2")));
+        OpenSecurityAnalysis securityAnalysis = new OpenSecurityAnalysis(network, new DefaultLimitViolationDetector(),
+            new LimitViolationFilter(), new DenseMatrixFactory(), () -> new NaiveGraphDecrementalConnectivity<>(LfBus::getNum));
+        SecurityAnalysisResult result = securityAnalysis.runSync(saParameters, contingenciesProvider);
+        assertTrue(result.getPreContingencyResult().isComputationOk());
+        assertEquals(0, result.getPreContingencyResult().getLimitViolations().size());
+        assertEquals(2, result.getPostContingencyResults().size());
+        assertTrue(result.getPostContingencyResults().get(0).getLimitViolationsResult().isComputationOk());
+        assertEquals(2, result.getPostContingencyResults().get(0).getLimitViolationsResult().getLimitViolations().size());
+        assertTrue(result.getPostContingencyResults().get(1).getLimitViolationsResult().isComputationOk());
+        assertEquals(4, result.getPostContingencyResults().get(1).getLimitViolationsResult().getLimitViolations().size());
```

Comment:
I could not be more than 2 :-) To be corrected. ````L2```` seems to be still in the network but should be out of load.